testdata/v1beta1/clustertask/pipelines/jib-maven-pz.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: jib-maven-pipeline
+spec:
+  params:
+  - name: SUBDIR
+    description: where to clone the git repo
+    default: jib-maven
+  workspaces:
+  - name: source
+  tasks:
+  - name: clone-git-repo
+    taskRef:
+      name: git-clone
+      kind: ClusterTask
+    workspaces:
+    - name: output
+      workspace: source
+    params:
+    - name: url
+      value: https://github.com/che-samples/console-java-simple
+    - name: subdirectory
+      value: $(params.SUBDIR)
+    - name: deleteExisting
+      value: "true"
+  - name: build
+    taskRef:
+      name: jib-maven
+      kind: ClusterTask
+    runAfter:
+    - clone-git-repo
+    workspaces:
+    - name: source
+      workspace: source
+    params:
+    - name: DIRECTORY
+      value: $(params.SUBDIR)
+    - name: IMAGE
+      value: image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/jib-maven
+    - name: INSECUREREGISTRY
+      value: "false"
+    - name: MAVEN_IMAGE
+      value: maven:3.6.3-adoptopenjdk-11
```

Comment:
let's update it to `maven:3.8.3-adoptopenjdk-11` - it worked fine when I was testing 1.8.0